build.gradle

```diff
@@ -60,3 +70,29 @@ publishing {
 tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
 }
+
+tasks.named('test') {
+    useJUnitPlatform()
+}
+
+
+configurations {
+    cucumberRuntime {
+        extendsFrom testImplementation
+    }
+}
+
+task cucumberCli() {
```

Comment:
please update README.md with details about how to use this task

Reply:
noted

Reply:
@fynmanoj done, please recheck